src/clusterfuzz/_internal/tests/core/platforms/android/settings_data/test_config.json

```diff
@@ -0,0 +1,10 @@
+{
+	"auto_build_version_to_fuzz_path": {
+		"r": "gs://auto/git_r/something/something/%something%/([something]+).zip",
```

Comment:
Can this match environment.get_platform_id() completely `android:{codename}_{sanitizer}:{build_version}` instead.

Reply:
Yes, that is probably better, and easy to generalize for the auto, phone-kernelspace and phone-userspace as well. Will get to work...